server/services/premisesService.ts

```diff
@@ -131,6 +143,41 @@ export default class PremisesService {
     }
   }
 
+  private async temporaryAccommodationSummaryListForPremises(premises: Premises): Promise<SummaryList> {
+    return {
+      rows: [
+        {
+          key: this.textValue('Property name'),
+          value: this.textValue(premises.name),
+        },
+        {
+          key: this.textValue('Address'),
+          value: this.htmlValue(`${escape(premises.address)}<br />${escape(premises.postcode)}`),
```

Comment:
I may be missing something here but couldn't we do `value.html: this.htmlValue(...` and avoid having to add the `escape` function? https://design-system.service.gov.uk/components/summary-list/#options-summary-list-example--rows

Reply:
We want the `<br />` interpreted as HTML, but we don't want anything inside `address` or `postcode` interpretted as HTTML, hence we need to escape the latter but not the former